### ISSUE TITLE
Can't add accounts to my (new) Yubikey

### ISSUE THREAD

**Issue by m-ocean-it** [Yubico/yubioath-desktop#784]
- Yubico Authenticator version: `yubioath-desktop-5.1.0-linux.AppImage`
- Operating system and version: `Linux Mint 20.3 x86_64 (Kernel: 5.13.0-27-generic)`
- YubiKey model and version: `5C NFC (firmware: 5.4.3)`
- Bug description summary: `qml: addCredential failed: No eligiable connections are available ([<class 'yubikit.core.smartcard.SmartCardConnection'>])`

Steps to reproduce and exceptions thrown
---

Launch the AppImage (as root just in case): `sudo ./yubioath-desktop-5.1.0-linux.AppImage`.
Those messages appear right away:
```
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
WARNING: PC/SC not available. Smart card protocols will not function.
```
Enter the Yubikey. This message appears:
```
qml: calculateAll failed: No eligiable connections are available ([<class 'yubikit.core.smartcard.SmartCardConnection'>]).
```
In the GUI, press "Add account", enter the secret key (TOTP seed) and account name, then press "Add account". It doesn't work. In the GUI, it's `Unknown error`, and in the terminal:
```
qml: addCredential failed: No eligiable connections are available ([<class 'yubikit.core.smartcard.SmartCardConnection'>]).
```

**Comment by fdennis** [comment]
The message says that PCSC is not running. Could you verify that?

**Comment by m-ocean-it** [comment]
> The message says that PCSC is not running. Could you verify that?

Wow, didn't know about that thing at all. Fixed it.

For others with the same problem:

I first installed `pcsc-tool` and used `pcsc_scan` to scan for smart cards. It returned an error. Then I figured I needed to install `pcscd` and run it as a daemon (`sudo apt install pcscd && sudo systemctl start pcscd`). After that, `pcsc_scan` returns proper info and Yubico Authenticator works just fine.

Thanks! Closing the issue.
